packages/firestore/test/integration/api_internal/bundle.test.ts

```diff
@@ -62,6 +63,25 @@ apiDescribe('Bundles', (persistence: boolean) => {
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
       (db as any)._databaseId as DatabaseId
     );
+
+    builder.addNamedQuery(
+      'limit',
+      { seconds: 1000, nanos: 9999 },
+      (collectionReference('coll-1')
+        .orderBy('bar', 'desc')
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        .limit(1) as any)._query
+    );
```

Comment:
It would be so nice if these tests lived in "api", which would mean that they get tested via firestore-exp and the minified builds. Can we use pre-encoded bundle and load it from JSON:?